frontend/src/pages/auth/components/sign-in-form/sign-in-form.tsx

```diff
@@ -19,6 +19,10 @@ const SignInForm: React.FC<Properties> = ({ onSubmit }) => {
     validationSchema: userSignInValidationSchema,
   });
 
+  const { isLoading } = useAppSelector(({ auth }) => ({
+    isLoading: auth.authenticatedUserDataStatus === DataStatus.PENDING,
```

Comment:
we should have split loading statuses for authenticated user and sign-up sign-in, create another status like authDataStatus and set it in sign-up /sign-in actions in store
```suggestion
    isLoading: auth.authDataStatus === DataStatus.PENDING,
```

Reply:
sure, no problem